zip-0316.rst

```diff
@@ -3,11 +3,556 @@
+Unified Address format is likely to be incorporated into such schemes as a new
+address type.
+
+Concepts
+--------
+
+Wallets follow a model *Interaction Flow* as follows:
+
+1. A Recipient *generates* an Address.
+2. The Recipient wallet or human user encodes the address, and
+   *distributes* this Address Encoding through mechanisms which may be
+   “out-of-band” (example: they spray paint a QR Code on a sign) or
+   they may be more or less “in-band” (example: they include a string
+   encoding of a ``Reply-To`` address in an encrypted memo following a
+   common standard).
+3. A Sender wallet or user *imports* the Address Encoding through any of
+   a variety of mechanisms (QR Code scanning, Payment URIs, cut-and-paste,
+   or “in-band” protocols like ``Reply-To`` memos). This includes decoding
+   and validity checks.
+4. (Perhaps later in time) the Sender wallet executes a transfer of ZEC
+   (or other assets or future protocol state changes) to the Address.
+
+These steps are a funnel: encodings of the same Address may be distributed
+zero or more times through different means. Zero or more Senders may import
+addresses. Zero or more of those may execute a Transfer. A single Sender may
+execute multiple Transfers over time from a single import.
+
+[TODO: examples]
+
+Addresses
+---------
+
+A Unified Address (or UA for short) combines one or more Receivers.
+
+When new Transport Protocols are introduced to the Zcash protocol after
+Unified Addresses are standardized, those should introduce new Receiver Types
+but *not* different address types outside of the UA standard. There needs
+to be a compelling reason to deviate from the standard, since the benefits
+of UA come precisely from their applicability across all new protocol
+upgrades.
+
+Receivers
+---------
+
+Every Wallet must anticipate and properly parse a UA with any unknown
+arbitrary Receiver Type.
+
+When Transferring to a valid UA, a Sender must behave as if any unknown
+Receiver Type is simply not present for the purposes of the transfer.
+
+A Wallet may process unknown Receiver Types by indicating to the user
+their presence or similar information for usability or diagnostic purposes.
+
+Transport Encoding
+------------------
+
+The string encoding is “opaque” to human readers: it does *not* allow
+visual identification of which Receivers or Receiver Types are present.
+
+- Rationale: The general thinking behind UAs is to allow wallets to
+  streamline user experience (UX). If human users can parse a UA and
+  alter their behaviour based on that, then different users will end up
+  using the same wallet very differently; this complicates troubleshooting
+  and learning from other users or educational resources. Note that this
+  does not preclude a wallet from providing user-friendly displays or
+  indications about Receiver support, and the wallet's UX design can
+  decide when and how to do this and build a behavioural flow around that.
```

Comment:
Not sure if this rationale is persuasive, we ZIP editors were not persuaded. 😀

Reply:
We decided to just remove this rationale paragraph for now. cc: @nathan-at-least